appsensor-core/src/main/java/org/owasp/appsensor/core/IAppsensorEntity.java

```diff
@@ -0,0 +1,14 @@
+package org.owasp.appsensor.core;
+
+import java.io.Serializable;
+
+/**
+ * Interface denoting methods required to be provided by appsensor entity objects
+ * @author Maik Jäkel(m.jaekel@xsite.de) http://www.xsite.de
+ */
+public interface IAppsensorEntity extends Serializable{
```

Comment:
Got it. That's helpful. I saw how it was used but didn't realize the _id mapping was not happening. That seems like an oversight from those tools, but there are likely reasons they did so that I don't understand.